### DISCUSSION TITLE
Error: No such keg

### DISCUSSION THREAD

**Discussion by antlep** [Homebrew/discussions#200] [Everyday usage] [answered]
Hello,

I've got a problem after installed then uninstalled Python@3.9 on my Mac (v11 - Big Sur)

now I've got this error with `brew list python` :
> Error: No such keg: /usr/local/Cellar/python@3.9

I've got Python@3.8 installed and it works as I want for now...

**Comment by SMillerDev** [comment, marked as answer]
`brew list python` expands the install paths of the installed `python` formula. Since `python@3.9` isn't installed this will fail since `python == python@3.9` in Homebrew.

**Reply by antlep** [reply]
So is it normal ?

Sorry I'm not a big expert of python night homebrew

Thanks,

**Reply by SMillerDev** [reply]
Yes, this is normal.

**Comment by aktaseren** [comment]
@antlep @SMillerDev I am having same issue. Also,

When I run ` brew doctor`

```
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Some installed formulae are missing dependencies.
You should `brew install` the missing dependencies:
  brew install python@3.9 zeromq

Run `brew missing` for more details.

```

How can I get rid of this warning?

**Reply by gromgit** [reply]
By doing exactly what `brew doctor` said:
```
  brew install python@3.9 zeromq
```
Alternatively, uninstall all the formulae that depend on each of the above formulae. To see the (probably LONG) list:
```
brew uses --recursive --installed python@3.9
brew uses --recursive --installed zeromq
```

**Comment by aktaseren** [comment]
```
brew uses --recursive --installed python@3.9
brew uses --recursive --installed zeromq
```
returned ipython. I deleted for now and the warning is gone. Thank you for this.
